client/lib/Tab.ts

```diff
@@ -21,10 +29,12 @@ import RequestGenerator, { getRequestIdOrUrl } from './Request';
 import AwaitedEventTarget from './AwaitedEventTarget';
 import CookieStorage, { createCookieStorage } from './CookieStorage';
 import Agent, { IState as IAgentState } from './Agent';
+import { getAwaitedPathAsMethodArg } from './SetupAwaitedHandler';
 
 const { getState, setState } = StateMachine<Tab, IState>();
 const agentState = StateMachine<Agent, IAgentState>();
 const awaitedPathState = StateMachine<any, { awaitedPath: AwaitedPath }>();
+const awaitedHandler = new AwaitedHandler(this, getState, setState);
```

Comment:
remove this?

Reply:
Oh, yeah. How could you see that didn't end up getting used?

Reply:
lint ;)